### PR TITLE
Update three-ways-to-substitute-a-substring-of-a-string.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
As there are two questions in the PQ and it is `type-in-the-gap`, we have to make the second one use code blocks as well.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/working-with-strings/three-ways-to-substitute-a-substring-of-a-string.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/working-with-strings/three-ways-to-substitute-a-substring-of-a-string.md?ref=bbe239b6f76baa3b51ec3387f69ecb71c9900672)

File: python/python-core/working-with-strings/three-ways-to-substitute-a-substring-of-a-string.md [hunk 1]
```diff
@@ -79,21 +79,23 @@ print(t
  .???(weather="sunny"))
 ```
 
-The fastest and easiest way of formatting strings is by using: ???
+Using `f-strings`, print "Hey Enki, how are you?":
+
+```python
+app = "Enki"
```
Comment (kapnobatai137) [inline python/python-core/working-with-strings/three-ways-to-substitute-a-substring-of-a-string.md:85]: let's use single quotes here (the pythonic way)

```suggestion
app = 'Enki'
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]
:rocket:

**Pull request merged**